packages/storefront-lwc/src/modules/commerce/router/router.js

```diff
@@ -11,7 +12,8 @@ const router = new UniversalRouter(
         {
             path: '/search/:query',
             action: location => {
-                return `<commerce-product-search-results query="${location.params.query}"></commerce-product-search-results>`;
+                var query = xss(`${location.params.query}`);
```

Comment:
Not sure you need `${location.params.query}` inside the xss... You might be able to just do location.params.query